oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-tools https://github.com/alfadil/server-tools.git 13.0-mig-sql_request_abstract
```

Comment:
@tonygalmiche you must merge this branch into server-tools OCA's one to have `sql_request_abrstract` available

Reply:
Thank you for the answer. It works perfectly. I did :
```
git clone https://github.com/alfadil/server-tools.git
cd server-tools/
git checkout 13.0-mig-sql_request_abstract
```

And I tested with this request : 

```
SELECT
  so.name as x_name,
  so.partner_id as x_partner_id,
  sol.product_id as x_product_id,
  sol.product_uom_qty as x_product_uom_qty,
  so.state as x_state
FROM sale_order so inner join sale_order_line sol on sol.order_id=so.id

```
Everything works fine. Thank you for this work